mpas_analysis/ocean/climatology_map_antarctic_melt.py

```diff
@@ -212,11 +291,38 @@ class RemapMpasAntarcticMeltClimatology(RemapMpasClimatologySubtask):
     landIceMask : xarray.DataArray
         A mask indicating where there is land ice on the ocean grid (thus,
         where melt rates are valid)
+
+    renameDict : dict
+        A dictionary use to rename variables in the climatology
     """
     # Authors
     # -------
     # Xylar Asay-Davis
 
+    def setup_and_check(self):
+        """
+        Figure out which variable(s) to remap
+        """
+        # Authors
+        # -------
+        # Xylar Asay-Davis
+
+        totalFlux = 'timeMonthly_avg_landIceFreshwaterFluxTotal'
+        frazilFlux = 'timeMonthly_avg_frazilIceFreshwaterFlux'
+        landIceFlux = 'timeMonthly_avg_landIceFreshwaterFlux'
+        if totalFlux in self.mpasClimatologyTask.allVariables:
+            # include the total and constituent fluxes
+            self.variableList = [totalFlux, landIceFlux, frazilFlux]
+            self.renameDict = {totalFlux: 'landIceFreshwaterFluxTotal',
+                               landIceFlux: 'landIceFreshwaterFlux',
+                               frazilFlux: 'frazilIceFreshwaterFlux'}
+        else:
+            # we only have the old name without the frazil flux
+            self.variableList = [landIceFlux]
+            self.renameDict = {landIceFlux: 'landIceFreshwaterFluxTotal'}
```

Comment:
```suggestion
            self.renameDict = {landIceFlux: 'landIceFreshwaterFlux'}
```
Isn't this the old name?

Reply:
I agree that this is a little confusing I will think about how to make this work.

Reply:
I changed the way plotting vs. observations is handled so that this renaming is no longer needed.  Thanks for pointing it out.